Gemfile

```diff
@@ -76,8 +49,9 @@ gem 'noid', '~> 0.8'
 
 group :test do
   gem "rspec-its"
-  gem "ci_reporter_rspec"
+  gem "ci_reporter_rspec", require: false
```

Comment:
Is there a better way to do this? https://github.com/ualbertalib/HydraNorth/blob/master/lib/tasks/spec.rake